src/pilot/templates/nic-configs/5_port/compute-ovs-dpdk-mode1.yaml

```diff
@@ -0,0 +1,255 @@
+# Copyright (c) 2016-2018 Dell Inc. or its subsidiaries.
```

Comment:
I think we should create a new nic-config for this instead of trying to add to an existing NIC config.